utils/tests_fetcher.py

```diff
@@ -216,6 +216,66 @@ def get_modified_python_files(diff_with_last_commit=False):
         return get_diff(repo, repo.head.commit, parent_commits)
 
 
+def get_diff_for_py_and_mdx_files(repo, base_commit, commits):
```

Comment:
just copies of existing functions with some modifications for doctest test files.